singer-io/tap-intercom
# v.0.0.3 Fix alpha test issues

# Description of change
Adjust `client.py` to use API `v.1.4`; fix issues w/ `conversations` and `admins` missing fields; fix `company_attributes` API version error; fix `write_bookmark` issue when scrolling.

# Manual QA steps
Ran singer-discover, singer-check-tap, target-stitch (initial load and incremental sync). No errors. Verified missing populated fields in target database.
 
# Risks
 Low - only Stitch and Bytecode IO using during alpha testing.
 
# Rollback steps
 Revert to v.0.0.2
